CI/Dockerfile

```diff
@@ -1,9 +1,43 @@
+# Svalinn DAGMC Dockerfile
+
+# build from repository root directory with command
+# docker build -t dagmc -f CI/Dockerfile .
+
+# Global ARGS set before the first build stage are accessable by all build stages
+ARG build_mw_reg_tests=OFF
+ARG EMBREE_BRANCH='v3.6.1'
+ARG geant4_version=10.5.1
 ARG UBUNTU_VERSION=18.04
+ARG MOAB_BRANCH=5.3.0
+ARG double_down=OFF
+ARG static_exe=OFF
+ARG ci_jobs=20
+
+# dependency_image_location defaults to using a local image for all the dependencies
+# This also allows one to specify a remote image from a container registry
+# The DAGMC container registry can be found here https://github.com/orgs/svalinn/packages
+ARG dependency_image_location=moab
```

Comment:
I don't think we need this anymore thanks to the multistage-build

Reply:
resolved by removed this arg from the dockerfile